### DISCUSSION TITLE
How about all contributors and interested parties join me and Kosinkadink in our discord for discussion of this tool and much much more.

### DISCUSSION THREAD

**Discussion by B34STW4RS** [Kosinkadink/ComfyUI-VideoHelperSuite#38] [Ideas]
Everyone is welcome to join, we are mainly focused on developing the animation side of Stable Diffusion to new unimagined heights.

https://discord.gg/sTGXmpDJ

**Comment by rhyswynn** [comment]
Hi, this discord invite seems to have expired, could you please share an updated one? Thank you!

**Comment by B34STW4RS** [comment]
Here you go, new discord link for all who want it: https://discord.gg/kEHZK695

**Reply by eldragon0** [reply]
Dead link again
